plugins/modules/ipam_host_info.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Infoblox Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: ipam_host_info
+short_description: Manage IpamHost
+description:
+    - Manage IpamHost
+version_added: 2.0.0
+author: Infoblox Inc. (@infobloxopen)
+options:
+    id:
+        description:
+            - ID of the object
+        type: str
+        required: false
+    filters:
+        description:
+            - Filter dict to filter objects
+        type: dict
+        required: false
+    filter_query:
+        description:
+            - Filter query to filter objects
+        type: str
+        required: false
+    tag_filters:
+        description:
+            - Filter dict to filter objects by tags
+        type: dict
+        required: false
+    tag_filter_query:
+        description:
+            - Filter query to filter objects by tags
+        type: str
+        required: false
+
+extends_documentation_fragment:
+    - infoblox.bloxone.common
+"""  # noqa: E501
+
+EXAMPLES = r"""
+    - name: "Create a Host"
+      infoblox.bloxone.ipam_host:
+        name: "{{ name }}"
+        tags:
+           tag1: "{{ tag1_value }}"
```

Comment:
Provide an actual tag value and not a placeholder

Reply:
Done